### ISSUE TITLE
[Bug] `Set Auto-Sync` is disable in Zotero 7

### ISSUE THREAD

**Issue by HoangMinhPhan** [windingwind/zotero-better-notes#1138]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest Zotero and the latest plugin?

- [X] I have confirmed I'm using the latest Zotero and the latest plugin

### Environment

- OS: Windows 10 22H2
- Zotero Version: 7.0.5
- Plugin Version: 2.0.8

### Describe the bug

I try to export my note to markdown with "auto-sync" to Obsidian. However, my checkbox of "Set Auto-Sync" is disable as this photo below:
![image](https://github.com/user-attachments/assets/d47d328c-f0f0-4d61-8ec3-f3a0af639ee9)

### Debug Output

[JavaScript Error: "[Scite Zotero] bootstrap: Started Zotero Scite"]

[JavaScript Error: "[Scite Zotero] bootstrap: onMainWindowLoad"]

[JavaScript Error: "uncaught exception: undefined"]

[JavaScript Error: "uncaught exception: undefined"]

[JavaScript Error: "Registered columns"]

[JavaScript Error: "Registered Scite section: scite-zotero-plugin-scite-ai-scite-zotero-plugin-pane"]

[JavaScript Error: "Passing a URI to Zotero.File.getContentsAsync() is deprecated -- use Zotero.HTTP.request() instead"]

[JavaScript Error: "Passing a URI to Zotero.File.getContentsAsync() is deprecated -- use Zotero.HTTP.request() instead"]

[JavaScript Error: "TypeError: this.editorView is undefined" {file: "resource://zotero/note-editor/editor.js" line: 6}]

[JavaScript Error: "Security Error: Content at moz-nullprincipal:{8e615362-c31c-4f96-8c10-16726bf191a6} may not load or link to chrome://betternotes/content/treeView.html."]

[JavaScript Error: "Passing a URI to Zotero.File.getContentsAsync() is deprecated -- use Zotero.HTTP.request() instead"]

[JavaScript Error: "Passing a URI to Zotero.File.getContentsAsync() is deprecated -- use Zotero.HTTP.request() instead"]

[JavaScript Error: "Debugging DOI  for item  and extra "]

[JavaScript Error: "TypeError: this.editorView is undefined" {file: "resource://zotero/note-editor/editor.js" line: 6}]

[JavaScript Error: "Passing a URI to Zotero.File.getContentsAsync() is deprecated -- use Zotero.HTTP.request() instead"]

[JavaScript Error: "Passing a URI to Zotero.File.getContentsAsync() is deprecated -- use Zotero.HTTP.request() instead"]

[JavaScript Error: "TypeError: this.editorView is undefined" {file: "resource://zotero/note-editor/editor.js" line: 6}]

[JavaScript Error: "TypeError: emptyListMessage is null" {file: "chrome://zotero/content/standalone/standalone.js" line: 687}]
ZoteroStandalone</this.updateAddonsPane/<@chrome://zotero/content/standalone/standalone.js:687:7

[JavaScript Error: "uncaught exception: undefined"]

[JavaScript Error: "[Scite Zotero] bootstrap: Shutting down"]

[JavaScript Error: "Error running bootstrap method 'shutdown' on scite-zotero-plugin@scite.ai"]

[JavaScript Error: "Zotero.Scite.unload is not a function" {file: "jar:file:///C:/Users/myusername/AppData/Roaming/Zotero/Zotero/Profiles/r7zgqkpv.default/extensions/scite-zotero-plugin@scite.ai.xpi!/bootstrap.js" line: 114}]

[JavaScript Error: "uncaught exception: undefined"]

No chrome package registered for chrome://betternotes/content/icons/favicon.png

appName => Zotero, version => 7.0.5 (x64), os => Windows 10.0 19045, locale => en-US, extensions => Scite Zotero Plugin (V7+) (2.0.2, extension, disabled), Better Notes for Zotero (2.0.8, extension, disabled)

**Comment by HoangMinhPhan** [comment]
find a way to do it

**Comment by lihuashan2007** [comment]
I came across the same issue. Can I ask how did you work it out? Thanks!

**Comment by HoangMinhPhan** [comment]
> I came across the same issue. Can I ask how did you work it out? Thanks!

Hi, I choose this option then the set Auto-Sync will appear again. See the attached photo
![image](https://github.com/user-attachments/assets/9b5353ce-c339-4c9f-8e4c-c839a731e142)

**Comment by HoangMinhPhan** [comment]
I think the guideline of betterNote lacks this introduction

**Comment by windingwind** [comment]
I don't think that should be included in the guideline. The radio and checkbox have tooltip text that explains how the export options work.
